### PR TITLE
Replace travis with Github actions

### PR DESCRIPTION
**Pull request by maartenvanvliet**
Cuts down on an external service and is better integrated with github.

### CODE REVIEW

**Comment by doomspork** [comment]
@maartenvanvliet my man! I _literally_ spot to @gemantzu about doing this very thing not an hour ago.Great minds think alike :tada:

**Comment by maartenvanvliet** [comment]
Haha nice!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ![Elixir Companies](https://user-images.githubusercontent.com/73386/33328317-e6e58c6e-d416-11e7-9a16-b60700db0a51.png)
 
-[![Build Status](https://travis-ci.org/beam-community/elixir-companies.svg?branch=master)](https://travis-ci.org/beam-community/elixir-companies)
+![](https://github.com/beam-community/elixir-companies/workflows/.github/workflows/elixir.yml/badge.svg?branch=master)
```
Comment (gemantzu) [inline README.md]: In the preview it is not displayed properly. Is it just something temporary?

File: .github/workflows/elixir.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - uses: actions/checkout@v1.0.0
+      - uses: actions/setup-elixir@v1.0.0
+        with:
+          otp-version: 21.1
+          elixir-version: 1.7.4
+      - name: Install Dependencies
```
Comment (gemantzu) [inline .github/workflows/elixir.yml]: Just a question: local.hex and local.rebar are already run during the setup?

File: README.md (hunk 1 above)
Comment (maartenvanvliet) [reply README.md]: Think so, the action has not yet run in the master branch of `beam-community/elixir-companies`. 

I got the badge url from https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow#example-using-a-workflow-file-path

File: .github/workflows/elixir.yml (hunk 2 above)
Comment (maartenvanvliet) [reply .github/workflows/elixir.yml]: Yes, see https://github.com/actions/setup-elixir/blob/master/action.yml where `actions/setup-elixir` refers to. Installing `local.hex` and `local.rebar` defaults to true.

Comment (gemantzu) [reply .github/workflows/elixir.yml]: Nice!

**Comment by gemantzu** [comment]
Ok @maartenvanvliet, let's wait for @doomspork to wake up and we can merge and see how it works in reality when we do! Thanks!

**Comment by doomspork** [comment]
I have awoken and merged :tada: I'll have some availability today to troubleshoot this if something comes up.

**Comment by doomspork** [comment]
https://github.com/beam-community/elixir-companies/commit/05501a399872f651f0e292094a82d8fa9ab1f5d6/checks?check_suite_id=311206851

**Comment by maartenvanvliet** [comment]
The badge also works :) https://github.com/beam-community/elixir-companies/workflows/.github/workflows/elixir.yml/badge.svg?branch=master

**Comment by gemantzu** [comment]
Perfect! Thanks @maartenvanvliet !

**Pull request merged**